src/lib/updates.js

```diff
@@ -17,27 +17,20 @@ export const createUpdate = async (files = [], channel, ts, user, text) => {
     ...files.map(async (file) => {
       const publicUrl = await getPublicFileUrl(file.url_private, channel, user);
       if (!publicUrl) {
-        await Promise.all([
-          react("remove", channel, ts, "beachball"),
-          react("add", channel, ts, "x"),
-          reply(channel, ts, t("messages.errors.filetype")),
-        ]);
-        return "error";
+        await reply(channel, ts, t("messages.errors.filetype"))
+        return;
       } else if (publicUrl.url.toLowerCase().endsWith("heic")) {
         await Promise.all([
-          react("remove", channel, ts, "beachball"),
-          react("add", channel, ts, "x"),
           postEphemeral(channel, t("messages.errors.heic"), user),
           app.client.chat.delete({
             token: process.env.SLACK_USER_TOKEN,
             channel,
             ts,
           }),
         ]);
-        return "error";
+        return;
       } else if (publicUrl.url === "big boy") {
         await Promise.all([
-          react("remove", channel, ts, "beachball"),
           reply(channel, ts, t("messages.errors.bigimage")),
```

Comment:
Just did it because presumably all successful messages have their thing removed somewhere? Don't fully understand the order of operations so I can put it back in

Reply:
It's been a while since I worked on this file, let me try to remind myself how it worked

Reply:
Actually does that branch even run? I see no references to "big boy" or even the strings "big" and "boy" anywhere else in the repo.